### ISSUE TITLE
What to do with infraspecific plant names in RINSE?

### ISSUE THREAD

**Issue by peterdesmet** [inbo/alien-species-checklist#56]
@timadriaens 

## The issue

In the RINSE dataset, infraspecific plant names are written as:

> Crocus vernus vernus

Rather than the correct:

> Crocus vernus subsp. vernus

For some of those, GBIF isn't 100% sure if the match was made correctly. e.g. should:

> Amaranthus hybridus hybridus

Be matched to:

> Amaranthus hybridus subsp. hybridus
> Amaranthus hybridus var. hybridus
> Amaranthus hybridus subvar. hybridus

It chose the most likely `Amaranthus hybridus subsp. hybridus` but indicated a 99% certainty.

## My proposal

I propose to **accept the match GBIF proposes**:

* If the proposed match also appears in the manual of plants. E.g. GBIF proposes a variety for `Amaranthus hybridus hybridus` and `Amaranthus hybridus var. hybridus` as the only infraspecific name in Plants.
* The species is listed in the manual of plants. E.g. GBIF proposes a variety for `Brassica napus napus`, but the whole species `Brassica napus` is listed in the manual, so that already includes any infraspecific taxon. We only have to be careful with synonyms here, as one infraspecific name (xyz) might be the synonym of another taxon (ab), while the species (xy) has nothing to do with xyz. That's probably very rare though.

**Comment by peterdesmet** [comment]
Agreed to follow my proposal

**Comment by peterdesmet** [comment]
@timadriaens, can we accept these non-well defined infraspecific taxa in rinse that have a GBIF match:

* [ ] uncertain rank, DOUBTFUL, but parent in rinse
* [ ] uncertain rank, but parent in rinse
* [ ] uncertain rank, SYNONYM, but accepted in rinse

**Comment by peterdesmet** [comment]
Note that this issue can be shelved if we decide to remove RINSE from scope (#69)

**Comment by peterdesmet** [comment]
Blocked by #69

**Comment by peterdesmet** [comment]
#69 rinse considered out of scope. `wontfix`
